index.html

```diff
@@ -59,6 +59,87 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        HTML allows the display of many media types through elements such as
+        &lt;img&gt;, &lt;picture&gt;, or &lt;video&gt;, but it does not provide
+        a native manner to directly consume 3D content. Embedding such content
+        within a page is comparatively cumbersome and relies on scripting the
+        &lt;canvas&gt; element. We believe it is time to put 3D models on equal
+        footing with other, already supported, media types.
+      </p>
+      <p>
+        There is a variety of solutions for displaying three-dimensional models
+        on the web, including general WebGL libraries and frameworks that have
+        the ability to parse and render models. There are also more
+        purpose-built libraries devoted to a simpler set of capabilities around
+        a single model, providing a simpler API to achieve this.
+      </p>
+      <p>
+        However, because these libraries constitute a layer of abstraction on
+        top of the browser, they are unreliable as a basis for browser- and
+        system-level accessibility capabilities, and may require the disclosure
+        of more information to the JavaScript-level API responsible for
+        rendering, such as the eye and head position of the user. While this is
+        necessary for complex experiences experiences such as those provided by
+        [[webxr]], it should not be necessary for the basic, spatial display of
+        an object in the context of a page.
+      </p>
+      <p>
```

Comment:
This paragraph should be the first paragraph of the introduction, I think.

Reply:
That and the language update should be reflected in the PR now!

Reply:
Right, but this should really move up... it's still not the first paragraph.

Reply:
This also seems to imply a lack of accessibility in, say, model-viewer, which I would hope is not what is intended.

Reply:
Definitely not intended.